### PR TITLE
feat(express): Create express server

### PR DESCRIPTION
**Pull request by KurokuTetsuya**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,11 @@
     "extends": "@hazmi35/eslint-config/typescript"
   },
   "dependencies": {
+    "@types/express": "^4.17.9",
```
Comment (Hazmi35) [inline package.json:33]: Please move this to devDependencies

File: src/structures/Server.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+export class ExpressServer {
+    public readonly app = express();
+    public constructor(public readonly client: Client) {}
+
+    public start(): void {
+        this.app.listen(this.client.config.serverPort, () => {
+            this.client.logger.info(`Express server listening to 0.0.0.0:${this.client.config.serverPort}`);
+        });
+        this.app.use(text({ type: "*/*" }));
+        this.app.post("/interactions", (request, response) => {
```
Comment (Hazmi35) [inline src/structures/Server.ts:17]: Where is / path? just send `{ status: 200, message: "Hello World" }` in JSON format

**Review by Hazmi35 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (KurokuTetsuya) [reply package.json:33]: sorry, I missed that lmao

File: src/structures/Server.ts (hunk 2 above)
Comment (KurokuTetsuya) [reply src/structures/Server.ts:17]: Okay

**Review by Hazmi35 — APPROVED** [review]

**Review by Allvaa — APPROVED** [review]

**Pull request merged**